pandas/tests/groupby/test_grouping.py

```diff
@@ -149,6 +149,18 @@ def test_grouper_index_types(self):
             df.index = list(reversed(df.index.tolist()))
             df.groupby(list("abcde")).apply(lambda x: x)
 
+    def test_grouper_period_index(self):
+        # GH 32108
+        periods = 2
+        index = pd.period_range(start="2018-01", periods=periods, freq="M")
+        period_serie = pd.Series(range(periods), index=index)
```

Comment:
just name this period_series

Reply:
right can you construct the initial index like

```
> index = pd.period_range(start="2018-01", periods=periods, freq="M")
index = pd.period_range(start="2018-01", periods=periods, freq="M", name='foo')
```

Reply:
The line on the current version is already:

```python
index = pd.period_range(
            start="2018-01", periods=periods, freq="M", name="Month"
        )
```